packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapHeatmapController.m

```diff
@@ -0,0 +1,159 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "GoogleMapHeatmapController.h"
+#import "JsonConversions.h"
+@import GoogleMapsUtils;
+
+@implementation FLTGoogleMapHeatmapController {
+  GMUHeatmapTileLayer *_heatmap;
+  GMSMapView *_mapView;
```

Comment:
nits: use OBJC property instead.

Reply:
Not sure what this means, but it looks like what the other controllers are doing

Reply:
I mean this :)

```objc
@interface FLTGoogleMapHeatmapController()

@property (nonatomic, strong) GMUHeatmapTileLayer *healmapLayer;

@end

```

Reply:
Done